hooks/post_gen_project.py

```diff
@@ -1,6 +1,22 @@
-#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
 print("""
-### IMPORTANT###
+### IMPORTANT ###
```

Comment:
Proposal:

```
### NEXT STEPS ###

Enable version control:

    cd {{cookiecutter.repo_name}}
    git init

Enable code sanity checks on every commit:

    pip install pre-commit    
    pre-commit install

Register your app NOW by making a pull request to the AiiDA Lab App Registry
on https://github.com/aiidalab/aiidalab-registry
See this link also for a full list of keys for your metadata.json file.
```

Reply:
Done.
However, I have modified it a bit further and now reads:

```shell
Congratulations!
You have now taken the first step in creating your new AiiDA Lab App:

    {{cookiecutter.app_title}}

### NEXT STEPS ###

1. Open your new AiiDA Lab App folder

    cd {{cookiecutter.repo_name}}

2. Enable version control

    git init

3. Enable code sanity checks on every commit (recommended)

    pip install -U pre-commit
    pre-commit install

4. Register your app NOW by making a pull request to the AiiDA Lab App Registry
on https://github.com/aiidalab/aiidalab-registry
See this link also for a full list of keys for your metadata.json file.
```